msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md

```diff
@@ -59,7 +59,29 @@ IPublicClientApplication app =
     .WithBroker(options)
     .Build();
 
-var authResult = await app.AcquireTokenInteractive(new List<string>() { "User.Read" }).ExecuteAsync();
+IAccount existingAccount = await FetchExistingAccountFromCache().ConfigureAwait(false);
+AuthenticationResult result = null;
+
+try
+{
+    // Try to sing-in the previously signed-in account
+    if (existingAccount != null)
+    {
+        result = await _pca.AcquireTokenSilent(scopes, existingAccount).ExecuteAsync();
+    }
+    // If it does not exist, try to sign in with the Windows account
```

Comment:
```suggestion
    // If it does not exist, try to sign in with the Windows account
```

maybe we can say the account that is already signed in to Windows (currently logged in user or something like that)